packages/url_launcher/url_launcher/README.md

```diff
@@ -137,57 +156,46 @@ final Uri emailLaunchUri = Uri(
   }),
 );
 
-launch(emailLaunchUri.toString());
+launchUrl(emailLaunchUri);
 ```
 
-**Warning**: For any scheme other than `http` or `https`, you should use the
-`query` parameter and the `encodeQueryParameters` function shown above rather
-than `Uri`'s `queryParameters` constructor argument, due to
-[a bug](https://github.com/dart-lang/sdk/issues/43838) in the way `Uri`
-encodes query parameters. Using `queryParameters` will result in spaces being
-converted to `+` in many cases.
-
-### Handling missing URL receivers
+### URLs not handled by `Uri`
 
-A particular mobile device may not be able to receive all supported URL schemes.
-For example, a tablet may not have a cellular radio and thus no support for
-launching a URL using the `sms` scheme, or a device may not have an email app
-and thus no support for launching a URL using the `mailto` scheme.
+In rare cases, you may need to launch a URL that the host system considers
+valid, but cannot be expressed by `Uri`. For those cases, alternate APIs using
+strings are available by importing `url_launcher_string.dart`.
```

Comment:
I did think about it; my hope is that having to find a separate import is already enough of a deterrent.

If that turns out not to be true and we see a lot of people filing issue because they blindly used this and ignored all the warnings about how it should rarely be used, we could deprecate that import file and add one with a scary name.